doc/userguide/rules/http-keywords.rst

```diff
@@ -66,7 +66,7 @@ http.cookie                    http_cookie              Both
 http.response_body             http_server_body         Response
 http.server                    N/A                      Response
 http.location                  N/A                      Response
-file.data                      file_data (*)            Response
+file.data                      file_data (*)            Both
```

Comment:
This table contains only fields that are response keywords. I think you should add it also to the table above listing the request keyword.

Reply:
indeed